.github/workflows/ci.yml

```diff
@@ -218,7 +218,8 @@ jobs:
             !:trino-kudu,
             !:trino-phoenix,!:trino-iceberg,!:trino-druid,
             !:trino-docs,!:trino-server,!:trino-server-rpm,
-            !:trino-test-jdbc-compatibility-old-server'
+            !:trino-test-jdbc-compatibility-old-server,
+            !:trino-bigquery'
```

Comment:
the exclusions here should be in the same order as the tests definitions are, but i see memsql is already out of place.

Reply:
memsql and bigquery are separate jobs. So I am not sure what you suggest here.

Reply:
order the exclusions (the `!...` lines) here in order of test appearance elsewhere -- the `test` matrix, and dedicated test jobs (memsql, bq).